src/views/pages/models.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import ListenerComponent from '../../lib/listener-component.jsx';
+import ModelStore from '../../store/model-store.js';
+import CreateModelForm from '../components/create-model-form.jsx';
+import { getAllModels } from '../../lib/api-client'
+export default class Models extends ListenerComponent {
+
+    constructor() {
+        super();
+
+    }
+    getStores() {
+        return [
+            ModelStore
+        ];
+    }
+    getState() {
+        return {
+            models: ModelStore.getAllModels()
+        };
+    }
+    render() {
+        return (
+            <div id="models">
+                <h1>All models</h1>
+                {this.state.models.map(function(model, key){
```

Comment:
This is typically where I would create a component such as `Model` to separate things out a bit, and pass the model information to each Model.

e.g.

```javascript
{this.state.models.map((model, key) => {
    return <Model key={key} data={model} />;
})}
```